packages/ketcher-core/src/application/render/restruct/restruct.ts

```diff
@@ -296,9 +297,10 @@ class ReStruct {
   }
 
   clearVisel(visel: Visel): void {
-    visel.paths.forEach((path) => {
-      path.remove()
-    })
+    // TODO: do we need this? visel.clear() contains this.paths = []
```

Comment:
Does just setting it to empty array remove it from canvas though?

Reply:
Right, it doesn't, moved back